### PR TITLE
[framework] Formatting: utils, object, tx_context, types, random 5/N

### PR DESCRIPTION
**Pull request by damirka**
## Description 

Continues the formatting.

## Test plan 

Tests must pass, framework must be compatible.

---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:
- [ ] REST API:

### CODE REVIEW

File: crates/sui-framework/packages/sui-framework/tests/display_tests.move [hunk 1]
```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module sui::display_tests {
```
Comment (tnowacki) [inline crates/sui-framework/packages/sui-framework/tests/display_tests.move:5]: Should this be
```suggestion
// Copyright (c) Mysten Labs, Inc.
// SPDX-License-Identifier: Apache-2.0

#[test_only]
module sui::display_tests;
```

File: crates/sui-framework/packages/sui-framework/tests/hex_tests.move [hunk 2]
```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[test_only]
+module sui::hex_tests {
```
Comment (tnowacki) [inline crates/sui-framework/packages/sui-framework/tests/hex_tests.move:5]: ditto
```suggestion
#[test_only]
module sui::hex_tests;
```

**Review by tnowacki — APPROVED** [review]

File: crates/sui-framework/packages/sui-framework/tests/display_tests.move (hunk 1 above)
Comment (damirka) [reply crates/sui-framework/packages/sui-framework/tests/display_tests.move:5]: I haven't touched tests yet! So this is intentional.

File: crates/sui-framework/packages/sui-framework/tests/hex_tests.move (hunk 2 above)
Comment (damirka) [reply crates/sui-framework/packages/sui-framework/tests/hex_tests.move:5]: Ditto

**Pull request merged**